primitives/runtime/src/generic/unchecked_extrinsic.rs

```diff
@@ -330,7 +365,9 @@ mod tests {
 	const TEST_ACCOUNT: TestAccountId = 0;
 
 	// NOTE: this is demonstration. One can simply use `()` for testing.
-	#[derive(Debug, Encode, Decode, Clone, Eq, PartialEq, Ord, PartialOrd)]
+	#[derive(
+		Debug, Encode, Decode, Clone, Eq, PartialEq, Ord, PartialOrd, TypeInfo,
+	)]
```

Comment:
I bet cargo fmt is complaining about this:

```suggestion
	#[derive(Debug, Encode, Decode, Clone, Eq, PartialEq, Ord, PartialOrd, TypeInfo)]
```